src/components/Search/Search.tsx

```diff
@@ -24,8 +25,9 @@ export const Search: React.FC<getMarkets> = ({ getMarkets }) => {
     e: React.MouseEvent<HTMLButtonElement, MouseEvent>
   ) => {
     e.preventDefault();
-    getMarkets(zip);
+    getMarkets(zip, distance);
     setZip('');
+    setDistance(0);
```

Comment:
Why are we initializing distance to 15 but then resetting it to 0?

Reply:
Good catch! I made a note of this, then totally forgot. Thanks!